sources/packages/backend/apps/test-db-seeding/src/db-seeding/02-institution/05-manage-designations.ts

```diff
@@ -0,0 +1,57 @@
+import { Injectable } from "@nestjs/common";
+import { InjectRepository } from "@nestjs/typeorm";
+import { DesignationAgreement, InstitutionLocation } from "@sims/sims-db";
+import {
+  createFakeDesignationAgreement,
+  createMultipleFakeInstitutionLocations,
+} from "@sims/test-utils";
+import { Repository } from "typeorm";
+import {
+  DataSeed,
+  DataSeedMethod,
+  SeedPriorityOrder,
+} from "../../seed-executors";
+import { InstitutionHelperService } from "../../test-seed-helper-services";
+import { institutionUserName01 } from "../constants";
+
+@Injectable()
+@DataSeed("provider", SeedPriorityOrder.LastBatch)
+export class DesignationAgreementService {
+  constructor(
+    @InjectRepository(DesignationAgreement)
+    private designationAgreementRepo: Repository<DesignationAgreement>,
+    @InjectRepository(InstitutionLocation)
+    private institutionLocationRepo: Repository<InstitutionLocation>,
+    private institutionHelperService: InstitutionHelperService,
+  ) {}
+
+  /**
+   * Method to seed fake pending designation agreement.
+   */
+  @DataSeedMethod("method")
+  async createPendingDesignationAgreement(): Promise<void> {
+    const InstitutionAdminUser = institutionUserName01;
```

Comment:
Please adjust the const name to the correct case. It can also express better what it represents, for instance, `SIMS_COLL_F_ADMIN_USER`.